lib/matplotlib/backends/backend_pgf.py

```diff
@@ -400,9 +399,8 @@ def __init__(self, figure, fh, dummy=False):
         else:
             # if fh does not belong to a filename, deactivate draw_image
             if not hasattr(fh, 'name') or not os.path.exists(fh.name):
-                warnings.warn("streamed pgf-code does not support raster "
-                              "graphics, consider using the pgf-to-pdf option",
-                              UserWarning, stacklevel=2)
+                _log.warning("streamed pgf-code does not support raster "
```

Comment:
Isn't this of the category "things the that the user must change to stop the warning" and thus should be `_warn_external`?